### ISSUE TITLE
Date Log-Picture isn't equal Date Found-Log

### ISSUE THREAD

**Issue by letsiz** [cgeo/cgeo#11026]
I have found a cache and save this one as offline-log. After few days I write my log and add one or more pictures, the date of my picture(s) isn't equal with the log date itself. The picture became the date of the day, who I send my log online. I must adjust this later manually.

The picture should bear the date of my find, as is the case online at gc.com or oc.com.  is this possible to take into account or to implement?

Thank you

**Comment by Lineflyer** [comment]
Can you tell which picture date you refer to? Isnt it only the log which has an associated date?

**Comment by letsiz** [comment]
> Can you tell which picture date you refer to? Isnt it only the log which has an associated date?
Sorry, but I don't understand your question in detail.

Yes, the log itself has an associated date. The day where the cache was found and the logbook was signed. But an optional log image has also an associated date. Normally it is taken on the same day and the date of the picture should be equal to the log date of the cache. Within the gallery the date is used to sort all your pictures. Sort by date and the order you has find the caches or taken the pictures/collages/photos/illus for an earthcache for example.

If I log the cache after view days, the date of my found status is correct. But the image becomes the date of the day where I write and send my log.

**Comment by eddiemuc** [comment]
I looked onto the gc.com website, and there seem to be new options to edit "log image details", among them is a "date taken" and a flag "is waypoint". I never noticed these before, maybe they are relatively new? Anyway, this is afaik not supported by c:geo yet.

![image](https://user-images.githubusercontent.com/6909759/123602505-18edd380-d7f9-11eb-87b3-9e787cc7ac22.png)

**Comment by letsiz** [comment]
FYI This log image details exist for long time I think. But that's not what my request should be about.

The init "problem" is the default date of an log image. If you add a image on one of our logs at gc.com, the image log date is set to the log date itself. And yes, you can change it if you wish for.

The default in c:geo should be that the image log date is equal to the log date itself. Not the date i have send my log online after few days I have found and set the correct date of my found :)

**Comment by Lineflyer** [comment]
Thanks for further clarification.
That however looks like gc.com by default uses the upload date at this point and as c:geo does not support explicitly setting the date on upload it comes out with that default.

So fixing this might need implementation of that "picture taken" date anyhow.

**Comment by letsiz** [comment]
I just tested it again on gc.com. Sent a log online with image and a date in the past, the image also gets the date set before in the past. Not the current date of the day on which I have wrote the log.

It remains exciting ... thanks for the support and the great app!

**Comment by Lineflyer** [comment]
@letsiz You mean, you tested it using your browser instead of c:geo or is the problem gone in c:geo?

**Comment by letsiz** [comment]
@Lineflyer Yes, I have tested it within my desktop browser on gc.com, not with c:geo on my phone.

**Comment by Lineflyer** [comment]
A user on support mail also mentioned this problem (which also means its still existing):

Testing scenario is:
- Compose a log and attach an image to be sent with the log
- Set the log date to e.g. 7 days ago
- Post the log with image
- Check the log entry on the website

Result:
Log is posted for the selected date, however the image details of the attached image show the date, where the log has been sent to the server.
